babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -236,6 +236,41 @@ SqlCreateAttribute CreateTableAttributeJournalTable() :
     { return new SqlCreateAttributeJournalTable(id, getPos()); }
 }
 
+SqlCreateAttribute CreateTableAttributeJournal() :
+{
+  JournalType journalType = JournalType.UNSPECIFIED;
+  JournalModifier journalModifier = JournalModifier.UNSPECIFIED;
+}
+{
+    [
```

Comment:
Suggest wrapping these different clauses/cases in an outer parentheses and separated by |'s. There would be two benefits of this from a code readability/reasoning perspective:

1) IMO it's easier to reason about if there's only a single return statement in the end of the parsing function.
2) It would be easy to see at a quick glance which cases are mutually exclusive of one another.

Reply:
Yeah I like that better. It would make it a lot easier to read.